tests/snapd

```diff
@@ -99,12 +90,19 @@ done
 if [ "${ID}" != "ubuntu-core" ]; then
     (
         set -e
-        usermod -G lxd -a jenkins || true
+
+        test_user="testuser"
+        useradd -m ${test_user} || true
+        usermod -G lxd -a ${test_user}
```

Comment:
`useradd` supports adding additional groups via the `--groups` option.

Reply:
Did this to handle the case where the user already exists, but I suppose this should never happen. Updated to use -G with `useradd`.